### ISSUE TITLE
Update Data Observatory Analysis UI 

### ISSUE THREAD

**Issue by saleiva** [CartoDB/cartodb#9991]
@urbanphes can you upload the last mockups for this analysis after updating it with the changes?

**Comment by urbanphes** [comment]
@saleiva this is the mockup:

![01 layers - 06-100 analysis - enrich from do](https://cloud.githubusercontent.com/assets/14546701/19106622/a47ae57c-8ae8-11e6-9076-71edf9d0704b.png)

**Comment by saleiva** [comment]
thanks! cc/ @javierarce

**Comment by saleiva** [comment]
for adding context, we are having some issues on demos with the actual UI, since measurements are not well explained and is very hard to choose a name without knowing the data you are going to add. If possible, this should be done before releasing

**Comment by saleiva** [comment]
Having more issues when people not knowing where the data comes from (which is solved in this UI).

**Comment by saleiva** [comment]
rel w/ https://github.com/CartoDB/design/issues/432

**Comment by saleiva** [comment]
dupe of https://github.com/CartoDB/cartodb/issues/11333

**Comment by noguerol** [comment]
Design ready: https://invis.io/5G9YKGISB

vamos vamos vamos!

**Comment by talos** [comment]
Excited to see this moving along! Please ping me as you need me for implementation detail work.

**Comment by xavijam** [comment]
Closing in favor of new tickets created in order to implement this in parallel. Follow this https://github.com/CartoDB/cartodb/milestone/135.
